packages/fhir-converter/src/templates/cda/ValueSet/ImmunizationStatus.hbs

```diff
@@ -23,6 +23,8 @@
     "entered-in-error"
   {{else if (eq normalizedCode "error")}}
     "entered-in-error"
+  {{else if (eq normalizedCode "normal")}}
+    "complete"
```

Comment:
Fix for this issue https://metriport.slack.com/archives/C04T256DQPQ/p1712273301670799